config/admins.txt

```diff
@@ -149,3 +149,4 @@ WaylandSmithy = Game Master
 NamelessFairy = Game Master
 WalterMeldron = Game Master
 san7890 = Game Master
+flleeppyy = Game Master
```

Comment:
💀

Reply:
blud thought he was slick......

Reply:
ill give them the benefit of the doubt, maybe they were just doing it to test and forgot to remove it. it is their first pr, after all

Reply:
yeah i forgot to remove it i am so sorry

Reply:
Localhost automatically gives you max perms, so I don't think you even need to do this when testing.

Reply:
I do my development on Linux, so me connecting to my laptop is not localhost (10.0.0.30:1337)

Reply:
> I do my development on Linux, so me connecting to my laptop is not localhost (10.0.0.30:1337)

Scuffed.